### PR TITLE
[Naitei] Api Controller Bill

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/79593

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

https://github.com/user-attachments/assets/4fddc532-2f99-425b-a738-7d0f6c289dcc

### CODE REVIEW

**Comment by SuKay712** [comment]
ready

File: app/controllers/api/v1/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  def destroy
+    render json: {message: t("flash.log_out_successfully")}, status: :ok
+  end
+
+  private
+
+  def generate_jwt user
+    payload = {user_id: user.id, exp: 24.hours.from_now.to_i}
```
Comment (ghost) [inline app/controllers/api/v1/sessions_controller.rb]: 24 chuyển vào settings nhé

File: app/controllers/api/v1/bills_controller.rb [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+    render json: {error: t("flash.dont_have_permission")}, status: :forbidden
+  end
+
+  def create
+    @bill = @current_user.bills.build bill_params
+    if @bill.save
+      create_stripe_session @cart_details
+      transfer_data
+      # @user.send_bill_info @bill
```
Comment (ghost) [inline app/controllers/api/v1/bills_controller.rb]: ko dùng thì bỏ đi em

**Comment by SuKay712** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**